app/src/main/java/org/schabi/newpipe/about/LicenseFragmentHelper.kt

```diff
@@ -113,4 +114,34 @@ object LicenseFragmentHelper {
                 }
         }
     }
+    @JvmStatic
+    fun showLicense(context: Context?, component: SoftwareComponent): Disposable {
+        return if (context == null) {
+            Disposable.empty()
+        } else {
+            Observable.fromCallable { getFormattedLicense(context, component.license) }
+                .subscribeOn(Schedulers.io())
+                .observeOn(AndroidSchedulers.mainThread())
+                .subscribe { formattedLicense: String ->
+                    val webViewData = Base64.encodeToString(
+                        formattedLicense
+                            .toByteArray(StandardCharsets.UTF_8),
+                        Base64.NO_PADDING
+                    )
+                    val webView = WebView(context)
+                    webView.loadData(webViewData, "text/html; charset=UTF-8", "base64")
+                    val alert = AlertDialog.Builder(context)
+                    alert.setTitle(component.license.name)
+                    alert.setView(webView)
+                    Localization.assureCorrectAppLanguage(context)
+                    alert.setPositiveButton(
+                        "DISMISS"
+                    ) { dialog, _ -> dialog.dismiss() }
+                    alert.setNeutralButton("Open website") { _, _ ->
+                        ShareUtils.openUrlInBrowser(context, component.link)
+                    }
```

Comment:
Please use string resources instead of raw strings, so they can be translated.

Reply:
Done.